### ISSUE TITLE
db:migrate returns status code 0 when migration fails

### ISSUE THREAD

**Issue by renou** [sequelize/cli#110]
Migration failures on db level are not handled properly and the command returns success code 0 and prints

Using environment 'development'.
== 20150318142814-init: migrating =======
Possibly unhandled SequelizeDatabaseError: type "enum_Group_status" already exists

**Comment by sdepold** [comment]
It would be nice if you could create a test as a pull request that shows the failing behavior. Otherwise I'll try to take a look at it at some point.

**Comment by renou** [comment]
```javascript
var sh = require('execSync');
var result = sh.exec('node_modules/.bin/sequelize db:migrate');
if(result.code || result.stdout.indexOf('SequelizeDatabaseError') > -1) {
  logger.error('has SequelizeDatabaseError but return code is 0');
  throw new Error(result.stdout);
} else {
  applauseLogger.info(result.stdout);
}
```

**Comment by theazureshadow** [comment]
+1. We need to fail deployments that didn't migrate correctly, and without an exit code our deployment scripts can't tell that they failed.

**Comment by arturhtml** [comment]
+1

**Comment by sorin** [comment]
+1

**Comment by sdepold** [comment]
fixed in 1.7.4
